app/lib/context-helper.js

```diff
@@ -0,0 +1,18 @@
+import { config } from '../config/index.js'
+import { PI_HUNT_AND_DAIRY_FOLLOW_UP_RELEASE_DATE } from '../constants/index.js'
+
+export const isPIHuntEnabled = () => {
+  return config.optionalPIHunt.enabled
+}
+export const isPIHuntEnabledAndVisitDateAfterGoLive = (dateOfVisit) => {
+  if (!dateOfVisit) {
+    throw new Error('dateOfVisit must be provided')
+  }
```

Comment:
Looks like every time this is called, it's called with dateOfVisit and given the below code will error if that ever changes, not sure we need to be so defensive.